lint-staged.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 
   '**/package.json': 'node ./scripts/lint-staged/no-tslint-deps',
 
-  'packages/!(react-examples)/**/(docs|examples)/*': 'node ./scripts/lint-staged/no-old-example-paths',
+  'packages/!(react-examples)/**/(docs|examples)/!(*.txt)': 'node ./scripts/lint-staged/no-old-example-paths',
```

Comment:
This is because the previous regex was unintentionally triggered by the addition of `packages/tsx-editor/src/transpiler/examples/functionFabric.txt`